roles/common/defaults/main.yml

```diff
@@ -1,7 +1,10 @@
 ---
 # defaults file for common
 
-centos8_repos:
+common_log_directory: /var/log/ripu
```

Comment:
From what I remember, the `ripu` log file is something used for debugging when infra.leapp was created and Bob Mader was migrating his scripts to ansible. 

Maybe we can drop this as well in here, what do you think, @andywaltlova and @djdanielsson?

Reply:
yea you could probably remove it but if you wanted a place to try and push some extra logs or something you could leave it but I would rename it if you did

Reply:
alright, removed, if the need arises it can always be extended